src/main/webapp/feed.js

```diff
@@ -665,20 +683,21 @@ async function updateSigninStatus(isSignedIn) {
     const authorizeButton = document.getElementById('authorize-button');
     const signoutButton = document.getElementById('signout-button');
     const feedRightSide = document.getElementById("feed-right-side");
-if (isSignedIn) {
-    authorizeButton.style.display = 'none';
-    signoutButton.style.display = 'block';
-    feedRightSide.style.display = "block";  
-    await getUserInfo();
-    await loadPage();
-    showAddToCalendarButtons();
-} else {
-    authorizeButton.style.display = 'block';
-    signoutButton.style.display = 'none';
-    feedRightSide.style.display = "none";
-    await loadPage();
-    hideAddToCalendarButtons();
-}
+    if (isSignedIn) {
```

Comment:
If the change in this code block is only formatting (re-indentation in this case) - drop it from this PR and make another PR that contains only formatting changes. It makes it simpler to understand blame history.